converters/lance-image-dataset-converter/convert-any-image-dataset-to-lance.py

```diff
@@ -0,0 +1,106 @@
+import os
+import argparse
+import pandas as pd
+import pyarrow as pa
+import lance
+import time
+from tqdm import tqdm
+
+def process_images(images_folder, data_type):
+    # Define schema for RecordBatch
+    schema = pa.schema([('image', pa.binary()), 
+                        ('filename', pa.string()), 
+                        ('category', pa.string()), 
+                        ('data_type', pa.string())])
```

Comment:
"data_type" should be renamed to "split" throughout the script since it represents the train/test/val split.